BuildScripts/mvBESFunctions.sql

```diff
@@ -234,6 +235,48 @@ begin
         RAISE INFO      E'Error Context:% % \n % \n %',CHR(10),  tUpdatePgMviewsSqlStatement, tRenameTableSqlStatement, tRenameIndexSqlStatement;
         RAISE EXCEPTION '%',                SQLSTATE;
 
+END;
+$BODY$
+LANGUAGE    plpgsql
+SECURITY    DEFINER;
+------------------------------------------------------------------------------------------------------------------------------------
+CREATE OR REPLACE
+FUNCTION    mv$version()
+    RETURNS TEXT
+AS
+$BODY$
+/* ---------------------------------------------------------------------------------------------------------------------------------
+Routine Name: mv$help
+Author:       Rohan Port
+Date:         18/10/2021
+------------------------------------------------------------------------------------------------------------------------------------
+Revision History    Push Down List
+------------------------------------------------------------------------------------------------------------------------------------
+Date        | Name          | Description
+------------+---------------+-------------------------------------------------------------------------------------------------------
+            |               |
+------------+---------------+-------------------------------------------------------------------------------------------------------
+Description:    Displays the version
+
+Arguments:      IN      None
+Returns:                TEXT    The version 
+
+************************************************************************************************************************************
+Copyright 2021 Beyond Essential Systems Pty Ltd
+***********************************************************************************************************************************/
+DECLARE
+
+BEGIN
+
+    RETURN '1_0_0';
```

Comment:
Went with this ugly `_` naming syntax since we write the migrations to folders by version, eg.
`patches/migrations/1_0_0/564342-SomePatch.js`